### ISSUE TITLE
A11y_api-catalog_ System.Activities.Debugger.Symbol _ Screen Reader: There is no alternate text given for “Usage” graph (graph is given as image) present in the page and NVDA is reading the graph as clickable

### ISSUE THREAD

**Issue by SaisreeDandamudi** [terrajobst/apisof.net#10]
[Check out Accessibility Insights] - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHubTags:**
#A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_api-catalog; #MAS1.3.1; #Screen Reader; #C+E-H; #api-catalog; #Narrator; #NVDA; #Edge; #Chrome; #Benchmark; #A11ySev1;

**Environment Details:** 
URL: https://apisof.net/

**Browser:** 
Microsoft Edge 44.18362.267.0
Chrome Version 76.0.3809.100

**Screen Readers:**
Narrator
NVDA

**Re-pro Steps:**
1.	Start Narrator/NVDA
2.	Hit the [URL](https://apisof.net/)
3.	Tab till “Browse” button and press enter
4.	Select any of the Catalog present in the list
5.	Turn on the scan mode and navigate till “Usage” graph and verify whether screen reader is reading any information about the graph

**Actual Result:** 
There is no alternate text given for “Usage” graph (graph is given as image) present in the page, so screen reader is not reading any information about the graph. Screen reader is just reading as “Image” and NVDA is reading the graph as “graphic clickable” 

**Expected result:** 
Since the graph is in “Image” format, there should be alternate text provided for the graph which explains the complete information about the graph. (I.e. the alternate text should explain about axis information and data information present in the graph). And NVDA should not read as clickable since the graph is not clickable

**User Impact:** 
Screen Reader user will not get to know any information about the graph if there is no alternate text given for graph

![10 NVDA is not reading any information about graph and reading as clickable](https://user-images.githubusercontent.com/51435580/63573815-f9097b00-c5a3-11e9-8703-40d79672f291.png)
[10.Narrator is not reading any information about graph.pptx](https://github.com/terrajobst/apisof.net/files/3533747/10.Narrator.is.not.reading.any.information.about.graph.pptx)

**Comment by terrajobst** [comment]
The web site no longer exists.
